sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -3015,6 +3059,15 @@ export class BlockBlobClient extends BlobClient {
    * @param {BlockBlobUploadOptions} [options] Options to the Block Blob Upload operation.
    * @returns {Promise<BlockBlobUploadResponse>} Response data for the Block Blob Upload operation.
    * @memberof BlockBlobClient
+   *
+   * @example
+   * ```js
+   * const content = "hello";
+   * const blobName = "newblob" + new Date().getTime();
+   * const blobClient = containerClient.getBlobClient(blobName);
+   * const blockBlobClient = blobClient.getBlockBlobClient();
+   * const uploadBlobResponse = await blockBlobClient.upload(content, content.length);
```

Comment:
The lines [2,3,4] can be consolidated to 
> const blockBlobClient = containerClient.getBlobClient("blobName");

Reply:
same at some other places**

Reply:
This is the way the samples are given in the README. I can diverge from that for the sake of keeping the inline samples shorter, but I don't know if that's what we should do.

Reply:
I am ok doing the consolidation and touching the readme in a separate PR.

Reply:
We should recommend customer using the conveniencer way just like Harsha mentioned:

`const blockBlobClient = containerClient.getBlobBlockClient(blobName);`